src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -300,10 +300,10 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 
 		this.loadNewTabs(allTabs.filter((tab) => tab.group === homeTabGroupId));
 
-		// If preview features are disabled only show the home tab
+		// If preview features are disabled filter out contributions from preview extensions
```

Comment:
What does it mean for an extension to be preview? I thought we agreed as long as the extension is labeled as preview, since the user has to specifically download it, it can bypass the preview flag?

Reply:
I might be misremembering but I thought that we agreed that while customers do specifically install extensions with the preview flag we still wanted to respect the flag for features in those extensions. Let me follow up offline to double check this...

Reply:
hmm, good point here, maybe we need to scope this to the builtin extensions.